src/components/Radio/Radio.tsx

```diff
@@ -6,6 +6,9 @@ export type RadioProps = NativeAttributes<'input'> & {
   /** The label associated with the Radio. Appears on the right. */
   label?: string;
 
+  /** The size of the Radio. Defaults to `small`. */
```

Comment:
I think we have a docs typo?

```suggestion
  /** The size of the Radio. Defaults to `medium`. */
```

Reply:
Now we dont :)